crypto/fipsmodule/rand/urandom_test.cc

```diff
@@ -420,13 +419,14 @@ static std::vector<Event> TestFunctionPRNGModel(unsigned flags) {
     };
   }
 
-  const size_t kSeedLength = CTR_DRBG_ENTROPY_LEN * (is_fips ? 10 : 1);
+  const size_t kSeedLength = CTR_DRBG_ENTROPY_LEN;
   const size_t kAdditionalDataLength = 32;
 
   if (!have_rdrand()) {
     if ((!have_fork_detection() && !sysrand(true, kAdditionalDataLength)) ||
-        // Initialise CRNGT.
-        (is_fips && !sysrand(true, 16)) ||
+        // Initialise CRNGT. In FIPS mode we use a blocking sysrand call for
```

Comment:
As far as I understand you need the module to make a system call to be able to intercept it and model it in the urandom test. The only time when Jitter makes a system call is when `rdtsc` is not available on the platform then it will call `clock_gettime` function. So we can try to intercept that call on ARM for example. Is this what you had in mind?

Reply:
I think `rdtsc` is available on every x86 CPU and not available on others. So we can probe the CPU type to determine `rdtsc` availability. But I'd prefer to do all this in a new PR if you don't mind?